### PR TITLE
New article: Sitecore certification article

### PR DESCRIPTION
**Pull request by DonNateR**

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f8a6807a28/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-47e5d469ab/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-251630bae1/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-4f4402c383/how-to/). :bow:

**Comment by kmsholcomb** [comment]
Hi @DonNateR. Edits attached. I had a question or two about the steps. I don't know if you can verify them, or if you need the team to do it. Of course, it looked to me that most of the article was leveraged from the Microsoft docs, so perhaps that is verification enough? :) Anyway, let me know if you have any questions. Thanks!

[Uploading Upload an SSL certificate to a custom domain_editsKH.pdf…]()
[Upload an SSL certificate to a custom domain_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1012600/Upload.an.SSL.certificate.to.a.custom.domain_editsKH.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-7fec749bf9/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-1a12ce7309/how-to/).

File: content/managed-operations/upload-an-ssl-certificate-to-a-sitecore-custom-domain.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+type: article
+created_date: '2017-05-08'
+created_by: Juan Garza
+last_modified_date: '2017-05-08'
+last_modified_by: Nate Archer
+product: Managed Operations
+product_url: managed-operations
+---
+
+To secure a application that has a custom domain name, you need to use HTTPS. By default, Azure secures the **.azurewebsites.net** wildcard domain with a single SSL certificate, so you can already access your application using the  **https://\<appname>.azurewebsites.net** URL.
```
Comment (catlook) [inline content/managed-operations/upload-an-ssl-certificate-to-a-sitecore-custom-domain.md]: To secure an application ...

File: content/managed-operations/upload-an-ssl-certificate-to-a-sitecore-custom-domain.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+However, the default Azure SSL certificate does not work if you use a custom domain for your application . A custom domain with its own SSL certificate is more secure than the default. This article describes how to add an SSL certificate to a application  with a custom domain.
```
Comment (catlook) [inline content/managed-operations/upload-an-ssl-certificate-to-a-sitecore-custom-domain.md]: ... for your application. A ...   [ extra space before period ]
... to an application with a  ...  [a to an, and extra space ]

File: content/managed-operations/upload-an-ssl-certificate-to-a-sitecore-custom-domain.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+### Prerequisite
+
+- Your login credentials for the Azure portal. For information about how to log in to the Azure portal, see [Sitecore Cloud portals and account management](/how-to/sitecore-cloud-portals-and-account-management/)
```
Comment (catlook) [inline content/managed-operations/upload-an-ssl-certificate-to-a-sitecore-custom-domain.md]: [ add a period after the link? ]

File: content/managed-operations/upload-an-ssl-certificate-to-a-sitecore-custom-domain.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+### Get an SSL certificate
+
+If you do not already have SSL certificate, you need to get one from a trusted certificate authority (CA). The certificate must meet all the following requirements:
+
+- Signed by a trusted CA (no private CA servers).
+
+- Contains a private key.
+
+- Created for key exchange, and exported to a .PFX file.
```
Comment (catlook) [inline content/managed-operations/upload-an-ssl-certificate-to-a-sitecore-custom-domain.md]: Be created for key exchange and exported ...  [removed comma andverb tense needs to be consistent in the list.  I started with the sentence "The certificate should:" and made the tenses conform.  Should be signed, should contain and so on]

File: content/managed-operations/upload-an-ssl-certificate-to-a-sitecore-custom-domain.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+- Your login credentials for the Azure portal. For information about how to log in to the Azure portal, see [Sitecore Cloud portals and account management](/how-to/sitecore-cloud-portals-and-account-management/)
+
+### Get an SSL certificate
+
+If you do not already have SSL certificate, you need to get one from a trusted certificate authority (CA). The certificate must meet all the following requirements:
+
+- Signed by a trusted CA (no private CA servers).
+
+- Contains a private key.
```
Comment (catlook) [inline content/managed-operations/upload-an-ssl-certificate-to-a-sitecore-custom-domain.md]: Contain a ...   [Verb tense needs to be consistent in the list.  I started with the sentence "The certificate should:" and made the tenses conform.  Should be signed, should contain and so on]

File: content/managed-operations/upload-an-ssl-certificate-to-a-sitecore-custom-domain.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+### Prerequisite
+
+- Your login credentials for the Azure portal. For information about how to log in to the Azure portal, see [Sitecore Cloud portals and account management](/how-to/sitecore-cloud-portals-and-account-management/)
+
+### Get an SSL certificate
+
+If you do not already have SSL certificate, you need to get one from a trusted certificate authority (CA). The certificate must meet all the following requirements:
+
+- Signed by a trusted CA (no private CA servers).
```
Comment (catlook) [inline content/managed-operations/upload-an-ssl-certificate-to-a-sitecore-custom-domain.md]: Be signed by ... [Verb tense needs to be consistent in the list.  I started with the sentence "The certificate should:" and made the tenses conform.  Should be signed, should contain and so on]

File: content/managed-operations/upload-an-ssl-certificate-to-a-sitecore-custom-domain.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+- Contains a private key.
+
+- Created for key exchange, and exported to a .PFX file.
+
+- Uses a minimum of 2048-bit encryption.
```
Comment (catlook) [inline content/managed-operations/upload-an-ssl-certificate-to-a-sitecore-custom-domain.md]: Use a minumum ... [Verb tense needs to be consistent in the list.  I started with the sentence "The certificate should:" and made the tenses conform.  Should be signed, should contain and so on]

File: content/managed-operations/upload-an-ssl-certificate-to-a-sitecore-custom-domain.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+- Have a subject name that matches the custom domain it needs to secure. To secure multiple domains with one certificate, you need to use a wildcard name (for example, *.contoso.com*) or specify subjectAltName values.
+
+- Merged with all intermediate certificates used by your CA. Otherwise, you might experience irreproducible interoperability problems on some clients.
```
Comment (catlook) [inline content/managed-operations/upload-an-ssl-certificate-to-a-sitecore-custom-domain.md]: Be merged with ... [Verb tense needs to be consistent in the list.  I started with the sentence "The certificate should:" and made the tenses conform.  Should be signed, should contain and so on]

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c98b8a60b6/how-to/)

**Pull request merged**